mix.exs

```diff
@@ -100,7 +100,6 @@ defmodule NervesLivebook.MixProject do
       {:nx, "~> 0.8.0"},
       {:phoenix_pubsub, "~> 2.0"},
       {:pinout, "~> 0.1"},
-      {:progress_bar, "~> 3.0"},
```

Comment:
I would probably leave progress_bar in. I've seen it used in LiveViews here and there. Or it might get re-added with Bumblebee. I think they use it.